packages/terra-alert/src/Alert.jsx

```diff
@@ -144,9 +145,21 @@ const Alert = ({
     { 'body-narrow': isNarrow && (onDismiss || action) },
   );
 
+  const alertId = uuidv4();
+  const alertTitleId = `alert-title-${alertId}`;
+  const alertMessageId = `alert-message-${alertId}`;
+
+  const dismissButtonAriaDescribedBy = title || defaultTitle ? alertTitleId : alertMessageId;
```

Comment:
small nit if linter is ok with adding parens

```suggestion
  const dismissButtonAriaDescribedBy = (title || defaultTitle) ? alertTitleId : alertMessageId;
```

Reply:
Added [626632b](https://github.com/cerner/terra-core/pull/3791/commits/626632be7b45e86d3fbcfb883131b6ad800d6f7b)